spec/features/general_feedback_spec.rb

```diff
@@ -1,63 +0,0 @@
-require 'rails_helper'
```

Comment:
It is good that we have controller specs to test permutations faster and only in one place, but I would keep just the happy path of this feature spec as they are valuable.